js/framework/AnalyticsConstants.js

```diff
@@ -9,37 +9,356 @@ if (!OO.Analytics.EVENTS)
 {
   const EVENTS =
   {
-    //messages for main video
-    //TODO see if we need to distinguish between video and ad messages
+    /**
+     * @public
+     * @event Analytics#EVENTS#VIDEO_PLAYER_CREATED
+     * @description This message is sent when the player is first created but
+     * may not be ready to play video yet.
+     * @param {Array} paramArray Array of length 1, containing the original parameters
```

Comment:
does event jsdoc have field param?  I think in constants.js we describe them params in the description